stdlib/2/whichdb.pyi

```diff
@@ -0,0 +1,8 @@
+# Source: https://hg.python.org/cpython/file/2.7/Lib/whichdb.py
+
+from types import ModuleType
+from typing import Optional
+
+dbm = ...  # type: Optional[ModuleType]
+
+def whichdb(filename: str) -> Optional[str]: ...
```

Comment:
In Python 2, `filename` can be `unicode`, too. Use `Text` instead of `str`, for the argument? (The return value is always `str`)